packages/terra-date-time-picker/tests/wdio/date-time-picker-mobile-spec.js

```diff
@@ -18,7 +18,7 @@ Terra.describeViewports('Date Time Picker Twelve Hour Mobile', ['tiny'], () => {
       browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/twelve-hour/date-time-picker-value-12-hour-mobile');
 
       browser.click('//*[@id="datetimeValueProvided"]/div[2]/div/div[3]/button[1]');
-      Terra.it.validatesElement();
+      Terra.validates.element();
```

Comment:
Is this change required?

Reply:
Using `Terra.it.validatesElement();` inside an `it` block means that `Terra.it.validatesElement();` won't get ran, which is happening here. Didn't notice it until I was running the tests locally.